client/src/components/PhotoGrid.jsx

```diff
@@ -41,6 +41,11 @@ export default class PhotoGrid extends React.Component {
       });
   }
 
+  handleShowAllPhotosButtonClick(e) {
+    e.preventDefault();

```

Comment:
What is the default interaction for a click event on a regular button? What happens if line 45 is taken out?